source/computer/macos-setup.rst

```diff
@@ -1,9 +1,221 @@
+:最近更新日期: 2021-01-15
+
+.. note::
+
+   本节内容适用于 macOS Catalina（10.15）和 macOS Big Sur（11.x）。
+   对于其他 macOS 版本不一定适用。
+
+----
+
+安装系统
+--------
+
+第一次启动 Mac 电脑后，经过简单的设置，就得到了一个可供日常使用的 macOS 系统。
+
+设置的过程很直观，唯一需要注意的是，在为电脑创建一个账户时，账户名应避免中文和空格。
+
+macOS 系统的更新也十分简单。当有新版本发布以后，可以直接在“系统偏好设置”的“软件更新”中直接更新即可。
+
+.. table:: 近几年的 macOS 系统版本号
+   :align: center  
+
+   ==================== ====================  ======================
+    版本号               代号                  发布日期
+   ==================== ====================  ======================
+   OS X 10.11           El Capitan            2015 年 9 月 30 日
+   macOS 10.12          Sierra                2016 年 9 月 20 日
+   macOS 10.13          High Sierra           2017 年 9 月 25 日
+   macOS 10.14          Mojave                2018 年 9 月 24 日
+   macOS 10.15          Catalina              2019 年 10 月 7 日
+   macOS 11             Big Sur               2020 年 9 月 12 日
+   ==================== ====================  ======================
+
+系统工具
+--------
+
+经过简单设置后的 macOS 系统，尚不能满足日常科研与编程开发的需求，还需要做
+进一步的配置。
+
+Command Line Tools for Xcode
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`Xcode <https://developer.apple.com/cn/xcode/>`__ 是 macOS 下的集成开发环境（IDE），
+类似于 Windows 下的 `Microsoft Visual Studio <https://visualstudio.microsoft.com/>`__\ 。
+Command Line Tools for Xcode 是 Xcode 的一部分，其包含了常用的命令行开发工具，
+比如 C 编译器、\ ``make``\ 、\ ``git`` 等，是 macOS 下编程开发的必须软件。
+
+按下 :kbd:`Command+空格`\ ，搜索“Terminal”并按下 :kbd:`Enter` 键以打开 Terminal 应用。
+在 Terminal 中执行如下命令以安装 Command Line Tools for Xcode::
+
+   $ xcode-select --install
+
+在弹出的窗口中点击“Install”即可。
+
+此处安装的 Command Line Tools for Xcode 可能不是最新版。点击左上角的 Apple 图标，
+在“系统偏好设置”的“软件更新”中查看是否有相关更新。如果有，则升级到最新版。
+
 .. note::
 
-   尚未完成。
+   Command Line Tools for Xcode 会被安装到 :file:`/Library/Developer/CommandLineTools/`
+   目录，其提供的命令行工具位于 :file:`/Library/Developer/CommandLineTools/usr/bin` 目录，
+   包括 ``gcc``、``make``、``git`` 等。
+
+Homebrew
+^^^^^^^^
+
+`Homebrew <https://brew.sh/index_zh-cn.html>`__ 是 macOS 下最流行的第三方软件包管理器，
+类似于 Linux 系统下的 ``apt``、``yum``、``dnf`` 等包管理器。
+其可以用于安装各种常见的软件包、库文件以及字体。日常及科研工作中所需的大多数软件、
+库文件以及字体都可以通过 Homebrew 安装。
+
+安装
+""""
+
+要安装 Homebrew，需要将如下命令复制到终端中并执行::
+
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+Homebrew 会被安装到 :file:`/usr/local/` 目录下。通过 Homebrew 安装的所有软件包
+也都将被安装到该目录下。
+
+.. note::
+
+   GitHub 在国内访问不畅，以上安装命令可能会由于网络问题而失败。
+   若以上命令失败，可参考 https://github.com/ineo6/homebrew-install 的
+   解决办法。
+
+   同样由于网络的原因，Homebrew 在国内可能下载速度较慢。建议参照
+   https://github.com/ineo6/homebrew-install 教程设置中科大源或清华大学源。
+
+使用
+""""
+
+安装好 Homebrew 后，打开新的 Terminal 便可以使用 Homebrew 提供的 ``brew`` 命令。
+``brew`` 的详细用法见官方文档。此处仅列出一些常用的用法::
+
+    # 模糊搜索与 wget 相关的软件
+    $ brew search wget
+
+    # 安装 wget 软件包
+    $ brew install wget
+
+    # 安装 Visual Studio Code
+    # 由于 Visual Studio Code 是带图形界面的软件，因而这里需要使用 --cask 选项
+    $ brew install --cask visual-studio-code
+
+    # 升级某个软件
+    $ brew upgrade xxx
+
+    # 卸载某个软件
+    $ brew uninstall xxx
```

Comment:
As mentioned before, we need to add some notes about "brew, core, cask, bottles"

Reply:
I am working on it now.

Reply:
Great!